tutorials/multimodal_dapt_curation/curator/configs/text_semantic_dedupe_config.yaml

```diff
@@ -0,0 +1,30 @@
+# Configuration file for semdantic dedup
+cache_dir: "/home/ruchaa/projects/nvingest_curator_tutorial/NeMo-Curator/tutorials/multimodal_dapt_curation/code/workspace/semdedup_cache/text" #"workspace/semdedup_cache/text"
+num_files: 16
+
+# Embeddings configuration
+embedding_model_name_or_path: "sentence-transformers/all-MiniLM-L6-v2"
+embedding_batch_size: 128
+embeddings_save_loc: "embeddings"
+embedding_pooling_strategy: "mean_pooling"
```

Comment:
I think you can remove some of these parameters to match the other config file, or you could add them to the other config file.